scene/gui/popup_menu.cpp

```diff
@@ -888,6 +905,87 @@ void PopupMenu::_notification(int p_what) {
 					}
 				}
 			}
+
+			Input *input = Input::get_singleton();
+
+			if (input->is_action_just_released("ui_left") || input->is_action_just_released("ui_right") || input->is_action_just_released("ui_up") || input->is_action_just_released("ui_down")) {
+				gamepad_event_delay_ms = DEFAULT_GAMEPAD_EVENT_DELAY_MS;
+				set_process_internal(true);
+				return;
+			}
+
+			gamepad_event_delay_ms -= get_process_delta_time();
+			if (gamepad_event_delay_ms < 0) {
+				gamepad_event_delay_ms = gamepad_event_repeat_rate_ms;
```

Comment:
Theoretically, if you want a better consistent rate, you should probably use `gamepad_event_delay_ms = gamepad_event_repeat_rate_ms + gamepad_event_delay_ms;`.